cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandlerInternal.java

```diff
@@ -92,8 +94,13 @@ public void getAllAppDetails(HttpRequest request, HttpResponder responder,
     if (!namespaceQueryAdmin.exists(namespaceId)) {
       throw new NamespaceNotFoundException(namespaceId);
     }
-    responder.sendJson(HttpResponseStatus.OK,
-                       GSON.toJson(applicationLifecycleService.getApps(namespaceId)));
+
+    ScanApplicationsRequest.Builder builder = ScanApplicationsRequest.builder();
+    builder.setNamespaceId(namespaceId);
```

Comment:
since it's builder use call chaining (with new lines as it seems fit): `.builder().setNamespaceId(namespaceId).build()`